less/admin/ExtensionPage.less

```diff
@@ -142,11 +142,13 @@
 
   .ExtensionPage-permissions {
 
-    @media @phone {
-      > .container {
-        overflow-x: scroll;
-        padding-bottom: 20px;
-      }
+    .PermissionGrid-removeScope {
+      display: none;
+    }
+
+    > .container {
+      overflow-x: scroll;
+      padding-bottom: 20px;
```

Comment:
I solved it by selecting `PermissionGrid` instead of `container`. Let me know if this works for you.

Reply:
It's still the same unfortunately 😅, essentially anywhere you scroll the horizontal overflowing, it will automatically also affect the vertical one, from local tests, I believe what I proposed above might be the best way to approach this.

![flarum lan_admin (20)](https://user-images.githubusercontent.com/20267363/108131807-23ad9480-70b2-11eb-9ec9-8c48f2b183ea.png)

Reply:
Check now